src/app/pages/survey/survey.page.scss

```diff
@@ -17,6 +17,14 @@ ion-input {
   border-radius: 10px !important;
 }
 
+/* For the textarea input */
+ion-textarea {
+  background-color: var(--ion-color-secondary-tint);
```

Comment:
not really a priority right now but why do we need so many !importants here? What is overriding our styles? Can we use classes instead?

Reply:
we can use classed for them. But it's not important like you said. I preferred to use the actual ion-input and ion-textarea because it will apply to all the ion-input and textareas that are in the page and I wanted that.